src/tests/ftest/util/soak_utils.py

```diff
@@ -233,7 +254,7 @@ def get_daos_server_logs(self):
         self (obj): soak obj
     """
     daos_dir = self.outputsoak_dir + "/daos_server_logs"
-    logs_dir = os.environ.get("DAOS_TEST_LOG_DIR", "/var/tmp/daos_testing/")
+    logs_dir = self.test_env.log_dir + "/*log*"
```

Comment:
Better to let this handle the slashes
```suggestion
    logs_dir = os.path.join(self.test_env.log_dir, "*log*")
```